### PR TITLE
 added new funcionality/ Ver dimension

### PR DESCRIPTION
**Pull request by JankowskiArtur**

### CODE REVIEW

**Review by A-Gluszak — COMMENTED** [review]
OK

File: functionality_sending_a_parcel.py [hunk 1]
```diff
@@ -10,6 +10,34 @@ def get_package_dimensions():
...
+    if ht > 40:
+        print(
+            f"\n Przekoroczono makymalny rozmiar wysokości paczki o {round((ht - 40), 2)}cm\n"
+        )
+        return True
+    if dpt > 60:
+        print(
+            f"\n Przekoroczono makymalny rozmiar głebokości paczki o {round((dpt - 60), 2)}cm\n"
+        )
+        return True
```
Comment (siezyj) [inline functionality_sending_a_parcel.py]: return is_package_ok

File: functionality_sending_a_parcel.py [hunk 2]
```diff
@@ -10,6 +10,34 @@ def get_package_dimensions():
...
         print(
-            f"{e} - Proszę o wskazanie prawidłowej wartości w postaci cyfr.\nPodawanie wartości dziesiętnych powinno nastąpić po znaku '.' "
+            f"{e} - Proszę o wskazanie prawidłowej wartości w postaci cyfr.\nPodawanie wartości dziesiętnych powinno "
+            f"nastąpić po znaku '.'"
         )
         get_package_dimensions()
+
+
+def dimensions_validation(wt, dpt, ht, wd):
+    if wt > 20:
```
Comment (siezyj) [inline functionality_sending_a_parcel.py:21]: is_package_ok = True

File: functionality_sending_a_parcel.py [hunk 3]
```diff
@@ -10,6 +10,34 @@ def get_package_dimensions():
...
+        print(f"\n Przekoroczono makymalną wagę paczki o {round((wt - 20), 2)}kg\n")
+        return True
```
Comment (siezyj) [inline functionality_sending_a_parcel.py]: is_package_ok = False

**Pull request merged**